<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>620</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetMoveHistory">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>35</y>
      <width>240</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMoveHistory">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>5</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Move history</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelElapsedTime">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>5</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDiskAmount">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>5</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of disks (max XX)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDiskAmount">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>5</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>5</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFromPeg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>204</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>From peg</string>
    </property>
    <widget class="QRadioButton" name="fromLeftPeg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fromCenterPeg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fromRightPeg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxToPeg">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>204</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>To peg</string>
    </property>
    <widget class="QRadioButton" name="toLeftPeg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="toCenterPeg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="toRightPeg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonMove">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>224</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
